.eslintrc

```diff
@@ -1,3 +1,12 @@
 {
-  "extends": "hypothesis"
+  "extends": "hypothesis",
+  "rules": {
```

Comment:
These rules are all taken from the "eslint: <rule name>" mentioned in the appropriate section of the Airbnb guide. The one exception is `{ "destructuring": "all" }` which avoids warnings if you do:

```js
let { fieldWhichIsLaterModified, fieldWhichIsNotLaterModified } = object;
```

Instead of:

```js
const { fieldWhichIsNotLaterModified } = object;
let { fieldWhichIsLaterModified } = object;
```

I felt this was just a little bit too pedantic and more importantly ESLint can't yet fix such cases automatically.